docs/sources/monitor/monitor-docker-containers.md

```diff
@@ -0,0 +1,235 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/tutorials/scenarios/monitor-docker-containers/
+description: Learn how to use Grafana Alloy to monitor Docker containers
+menuTitle: Monitor Docker
+title: Monitor Docker containers with Grafana Alloy
+weight: 200
+---
+
+# Monitor Docker containers with {{% param "FULL_PRODUCT_NAME" %}}
+
+Docker containers provide statistics and logs.
+The `docker stats` and `docker logs` commands show the metrics and logs in a terminal as a fixed snapshot in time.
+You can use {{< param "PRODUCT_NAME" >}} to collect your metrics and logs, forward them to a Grafana stack, and create a Grafana dashboard to monitor your Docker container.
+
+The `alloy-scenarios` repository provides series of complete working examples of {{< param "PRODUCT_NAME" >}} deployments.
+You can clone the repository and use the example deployments to understand how {{< param "PRODUCT_NAME" >}} can collect, process, and export telemetry signals.
+
+In this example scenario, {{< param "PRODUCT_NAME" >}} collects the Docker container metrics and logs and forwards them to a Loki destination.
+
+## Before you begin
+
+This example requires:
+
+* Docker
+* Git
+
+{{< admonition type="note" >}}
+The `docker` commands require administrator privileges.
+{{< /admonition >}}
+
+## Clone and deploy the example
+
+Perform the following steps to clone the scenarios repository and deploy the monitoring example.
+
+1. Clone the {{< param "PRODUCT_NAME" >}} scenarios repository.
+
+   ```shell
+   git clone https://github.com/grafana/alloy-scenarios.git
+   ```
+
+1. Start Docker to deploy the Grafana stack.
+
+   ```shell
+   cd alloy-scenarios/docker-monitoring
+   docker compose up -d
+   ```
+
+   You can check the status of the Docker containers by running the following command.
+
+   ```shell
+   docker ps
+   ```
+
+1. (Optional) When you finish exploring this example, you can stop Docker to shut down the Grafana stack.
+
+   ```shell
+   docker compose down
+   ```
+
+## Monitor and visualize your data
+
+You can use Grafana to view the health of your deployment and visualize your data.
+
+### Monitor {{% param "PRODUCT_NAME" %}}
+
+To monitor the health of your {{< param "PRODUCT_NAME" >}} deployment, open your browser and navigate to [http://localhost:12345](http://localhost:12345).
+
+Refer to [Debug Grafana Alloy](https://grafana.com/docs/alloy/latest/troubleshoot/debug/) for more information about the {{< param "PRODUCT_NAME" >}} UI.
+
+### Visualize your data
+
+To explore metrics, open your browser and navigate to [http://localhost:3000/explore/metrics](http://localhost:3000/explore/metrics).
```

Comment:
This doesn't work, no Grafana on port 3000 for me.

I looked at the Docker compose logs, and starting Grafana fails with

```
Error: ✗ failed to download plugin archive: Get "https://storage.googleapis.com/integration-artifacts/grafana-lokiexplore-app/grafana-lokiexplore-app-latest.zip": dial tcp: lookup storage.googleapis.com on 127.0.0.11:53: server misbehaving
```

Reply:
Okay should now be fixed! Removed old install to Log Explore app